assignments/assignment01.rb

```diff
@@ -38,13 +38,36 @@ def number_to_string(n, lang)
 # it accepts a string
 # and returns the same string with each word capitalized.
 def titleize(s)
+  result = ""
```

Comment:
You're doing a lot of work here. Instead, try to use the built in ruby methods on String, such as split, capitalize, etc. Then you could solve this problem much more simply.